### DISCUSSION TITLE
Can you update to use latest windows app sdk, or latest .net 6 etc

### DISCUSSION THREAD

**Discussion by fisforfaheem** [vhanla/winxcorners#51] [General]
this will make the app look faster, more appealing and fresh. or you can use FLUTTER

**Comment by vhanla** [comment]
I tried WinUI3 (C#) and also C++ unpackaged WinUI3 to give it a modern look for Windows 10/11, but I was not convinced.
For instance, currently this one consumes 5MB in my computer, while WinUI3 more than 60MB, and to release the executable, it requires a lot of binaries for this to be published, too.
![imagen](https://github.com/vhanla/winxcorners/assets/1015823/c79c4a2c-ab76-406c-98c6-59d172ba14cd)
I tried Rust too, a similar issue, so much RAM usage for a simple tool.

I also tried FMX from Delphi, which looks like this.
![imagen](https://github.com/vhanla/winxcorners/assets/1015823/947e530e-e5de-43f6-9ee2-be33264d0aa6)
![imagen](https://github.com/vhanla/winxcorners/assets/1015823/e25aaaa4-e276-427b-95af-95b7dcf85f15)

But it consumes GPU and more RAM (119 MB) than current published one (VCL).

So, I decided to stick to this simple/basic one, which as of now, just consumes 5MB of RAM, and its executable is just a single one and 800KB.
![imagen](https://github.com/vhanla/winxcorners/assets/1015823/47746e02-9c0a-44da-8630-0ec6d3dab773)

![imagen](https://github.com/vhanla/winxcorners/assets/1015823/282c2842-ce8c-4b58-8d73-8bd7a6ea063c)
![imagen](https://github.com/vhanla/winxcorners/assets/1015823/a8835d82-4e84-4b60-a9f3-11a61ee152dc)

So basically, I was testing many other programming languages, I failed 😅. And now, I will focus back again to this one.
